playbooks/deploy_cluster.yml

```diff
@@ -1,44 +1,9 @@
 ---
 - import_playbook: init/main.yml
 
-- import_playbook: openshift-checks/private/install.yml
+- import_playbook: common/openshift-cluster/private/control_plane.yml
```

Comment:
We're trying to remove things from 'openshift-cluster'

We can't call this control_plane if it installs nodes, because our upgrade_control_plane only upgrades masters and etcd hosts.  Calling everything below pods the control plane is confusing.

Reply:
I think 'base_cluster' is better than 'control_plane'

I think the file path should be
common/private/base_cluster.yml

Reply:
I'm fine splitting out the nodes.  GCP and AWS will be using golden images so in general they wouldn't call this step either (I call it today in case people want to manage masters directly in an invocation).

Reply:
Actually, when we change how the control plane is installed (moving to static pods and self-hosted) we'll have to configure masters as nodes first.  So we shouldn't get attached to the current node order anyway.

For posterity, the arc we're on for control plane is going to be only supporting the following steps:

1. runc installed
2. kubelet installed inside system container
3. host level config that includes static pods
4. kubelet launches static pods, which include the control plane
5. static pods form working control plane
6. master nodes are bootstrapped into the cluster

The "bootstrap node" configuration will then be required as part of control plane setup (or required to have been completed).  So we should expect that some node config goes before the control plane, but that *all* nodes are configured in bootstrap mode (the current node config is going to be deprecated and removed).

I'm going to move the node setup into the callers for now.